docs/howtos/layers/labels.md

```diff
@@ -11,32 +11,334 @@ kernelspec:
+  `opaque` from the dropdown. Refer to the [Blending layers](blending-layers)
+  section of _Layers at a glance_ for an explanation of each type of blending.
+
+* Color mode
+
+  Select `auto` or `direct` from the dropdown. Auto is the default and allows
+  color to be set via a hash function with a seed. Direct allows the color of
+  each label to be set directly by a color dictionary.
+
+* Contour
+
+  If this combo box has any value other than 0, only the contours of the labels
+  will show. Change the value by clicking the - or + symbols on either end of
+  the bar.
+
+* n edit dim
+
+  This is the number of dimensions across which labels will be edited.
+
+* Contiguous
+
+  If this box is checked the `fill bucket` changes only connected pixels of the
+  same label.
+
+* Preserve labels
+
+  If this box is checked, existing labels are preserved while painting. It
+  defaults to false to allow painting on existing labels. When set to true,
+  existing labels will be preserved during painting. You can toggle
+  this mode using the default keybinding `p`.
+
+* Show selected
+
+  When this is checked only the selected labels will be displayed. Selected
+  labels are those labels that match the color in the `label` combo box. When it
+  is not checked, all labels will be displayed.
+
+## Editing using the tools in the GUI
+### Pan and zoom mode
+
+The default mode of the `labels layer` is to support panning and zooming. This
+mode is represented by the magnifying glass in the `layer controls` panel. While
+pan and zoom is selected editing the layer is not possible. Once you click on
+one of the editing tools, pan and zoom is turned off. Return to pan and zoom
+mode by pressing the `4` key when the `labels layer` is selected.
+
+### Creating a new `labels layer`
+
+Create a brand-new empty `labels layer` by clicking the `New labels layer`
+button above the layer list. The shape of the new labels layer will match the
+size of any currently existing image layers, allowing you to paint on top of
+them.
+
+### Selecting a label
+
+A particular label can be chosen in one of three ways:
+
+* Using the label combobox inside the `layer controls` panel and typing in the
+  numeric value of the desired label 
+
+* Using the + or - buttons to get to the desired label color or press the default 
+keybinding `m` to set a new label.
+
+* Selecting the `color picker` tool and then clicking on a pixel with the
+  desired label color in the image
+
+When a label is chosen its integer appears inside the label combobox and the
+color of the label is shown in the thumbnail next to the label combobox. If the
+0 label is selected, then a checkerboard pattern is shown in the thumbnail to
+represent the transparent color.
+
+You can quickly select the color picker by pressing the `4` key when the labels
+layer is selected.
+
+You can set the selected label to 0, the background label, by pressing `b`.
```

Comment:
```suggestion
While painting with a label, you can swap the current (selected) label to the transparent background label (`0`) by pressing `x`. Pressing `x` again will restore the last used label.
```
How does this sound? Trying to match the wording above.

Reply:
Another simpler option
> While painting with a label, you can toggle between the current (selected) label and the transparent background label (`0`) by pressing `x`.

or

> While painting with a label, you can swap between the current (selected) label and the transparent background label (`0`) by pressing `x`.

Reply:
Accepted your third option!